### PR TITLE
Generate OSGi metadata for drool-pmml

### PR DESCRIPTION
**Pull request by ctron**

### CODE REVIEW

File: drools-pmml/pom.xml [hunk 1]
```diff
@@ -26,6 +26,7 @@
   </parent>
 
   <artifactId>drools-pmml</artifactId>
+  <packaging>bundle</packaging>
```
Comment (mariofusco) [inline drools-pmml/pom.xml:29]: Afaik drools-ppml is no longer supported. @lanceleverich can you please confirm?

File: drools-pmml/pom.xml [hunk 2]
```diff
@@ -84,6 +91,10 @@
 
   <build>
     <plugins>
+      <plugin>
+        <groupId>org.apache.felix</groupId>
+        <artifactId>maven-bundle-plugin</artifactId>
```
Comment (mariofusco) [inline drools-pmml/pom.xml:96]: Shouldn't you declare here the packages imported and exported by this module?

Comment (ctron) [reply drools-pmml/pom.xml:96]: Not necessarily. As the plugin will pick that up automatically. Only in the case something goes wrong with that you need to start declaring things.

**Comment by ghost** [comment]
Can one of the admins verify this PR? Comment with 'ok to test' to start the build.

**Comment by ghost** [comment]
Can one of the admins verify this PR? Comment with 'ok to test' to start the build.

**Comment by mariofusco** [comment]
drools-ppml is no longer supported

**Pull request closed without merging**